src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -391,6 +400,115 @@ public void createMissingClass(UnsolvedClass missedClass) {
     }
   }
 
+  /**
+   * This method capitalizes a string. For example, "hello" will become "Hello".
+   *
+   * @param string the string to be capitalized
+   * @return the capitalized version of the string
+   */
+  public static String toCapital(String string) {
+    return string.substring(0, 1).toUpperCase() + string.substring(1);
+  }
+
+  /**
+   * This method checks if a string is capitalized
+   *
+   * @param string the string to be checked
+   * @return true if the string is capitalized
+   */
+  public static boolean isCapital(String string) {
+    String capitalizedString = toCapital(string);
+    return string.equals(capitalizedString);
+  }
+
+  /**
+   * This method checks if a string is a class path.
```

Comment:
I don't think this comment is accurate. This method checks if a string _has the form_ of a class path: that is, whether it _could_ be a classpath. It does not actually check that the string does represent a real class.